.github/workflows/push.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           pip install pytest
           pip install pytest-cov
           python -m pytest --cov=./ --cov-report=xml
-      - name: Upload coverage to Codecov  
+      - name: Upload coverage to Codecov
```

Comment:
The first commit is `spec(flake8): permit black-style line breaks`, but only seems to delete some whitespace. Is there supposed to be a flake8 config change?

Reply:
Looks like I snuck this in by accident: https://github.com/primitybio/cellengine-python-toolkit/blame/a1566ebb51899eeffeebc63231386eb95967e552/.github/workflows/push.yml#L28